.github/workflows/phpstan.yml

```diff
@@ -0,0 +1,39 @@
+name: 'PHPStan Tests'
+on:
+  pull_request:
+    paths:
+      - 'src/**.php'
+      - '*.php'
+jobs:
+  phpstan:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout the repository
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 1000
+          submodules: recursive
+      # ------------------------------------------------------------------------------
+      # Set up PHP
+      # ------------------------------------------------------------------------------
+      - name: Configure PHP environment
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: 7.4
+      - name: Install vendor
+        run: composer i --ignore-platform-reqs
+      # ------------------------------------------------------------------------------
+      # Set up Wordpress
+      # ------------------------------------------------------------------------------
+      - name: Install WP CLI
+        run: curl -O https://raw.githubusercontent.com/wp-cli/builds/gh-pages/phar/wp-cli.phar
+      - name: Install WP Core
+        run: php wp-cli.phar core download --path=wpdemo.test --allow-root
+      # ------------------------------------------------------------------------------
+      # Run the PHPStan tests
+      # ------------------------------------------------------------------------------
+      - name: Convert PHPStan Config for Local Paths
+        run: |
+          sed -i 's/..\/..\/..\/wp-includes/wpdemo.test\/wp-includes/g' phpstan.neon

```

Comment:
This looks a bit contrived: I suggest the creation of a `phpstan.ci.neon` file with the correct path and its use in the workflow.

Reply:
@lucatume I don't disagree, I considered that. 

The reason I went this route was because my concern was that we would basically be copying an entire file - where all changes would have to be duplicated between a local `neon` versus a CI environment `neon` file, all for one value that needs to be changed. As I understand it there is no "merge" of values, so we can't simply have an extra config that only has that value.

Reply:
Resolving as discussed in 2023-09-11 Dev meeting.